runtime/menu.vim

```diff
@@ -80,6 +80,9 @@ if has("gui_macvim")
   an <silent> 9998.350 Window.Select\ Previous\ Tab :tabprevious<CR>
   an 9998.360 Window.-SEP2-			    <Nop>
   an <silent> 9998.370 Window.Bring\ All\ To\ Front <Nop>
+  an <silent> 9998.380 Window.Stay\ On\ Bottom <Nop>
```

Comment:
iOS simulator has "Stay in Front" but no "Stay in Back". I can't find similar functionality elsewhere. Afloat (the SIMBL plugin) used "Keep Afloat".

Do you think "Stay in Front" and "Stay in Back" would be better names?